File: Curso_logica_de_programacao/Exercicios_logica_em_C#/Retangulo/Retangulo.cs
using System;
using System.Globalization;

namespace Progama
{
    class Progam
    {
        static void Main(string[] args)
        {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double base_retangulo, altura, area;

            Console.Write("Base retangulo: ");
            base_retangulo = double.Parse(Console.ReadLine(), CI);
            Console.Write("Altura retangulo: ");
            altura = double.Parse(Console.ReadLine(), CI);

            area = base_retangulo * altura;
            double perimetro = 2 * (base_retangulo+altura);
            double diagonal =  Math.Sqrt(Math.Pow(base_retangulo, 2) + Math.Pow(altura, 2));

            Console.WriteLine("Area = " + area.ToString("F4", CI));
            Console.WriteLine("Perimetro = " + perimetro.ToString("F4", CI));
            Console.WriteLine("Diagonal = " + diagonal.ToString("F4", CI));

        }
    }
}

File: Algoritmos_e_Logica_de_Programacao/Exercícios_logica_em_Python/Retangulo/Retangulo.py
import math

base: float
altura: float
area: float
diagonal: float

print("Digite a base do retangulo: ", end="")
base = float(input())

print("Digite a altura do retangulo: ", end="")
altura = float(input())


area = base * altura
perimetro: float
perimetro = 2 * (base + altura)
diagonal = math.sqrt(pow(base,2) + pow(altura,2))

print(f"Area do retangulo = {area:.4f}")
print(f"Perimetro do retangulo = {perimetro:.4f}")
print(f"Diagonal do retangulo = {diagonal:.4f}")

